### PR TITLE
several displayed descriptions for data criteria needed to change to …

### PR DESCRIPTION
**Pull request by edeyoung**
…conform with QDM 5.0 changes. However, the codes themselves seemed like they should stay the same for backwards compatibility. this led to discrepencies in the testing. these are addressed in this pull request.

### CODE REVIEW

File: test/unit/hqmf/model/data_criteria_test.rb [hunk 1]
```diff
@@ -19,6 +19,16 @@ def test_fields_definitions
       # ignore custom fulfills and previous ordinal field values
       mismatches -= ["FLFS - FULFILLS", "ORDINAL - ORDINALITY"]
 
+      # ignore changes needed when converting to qdm 5.0
```
Comment (c-monkey) [inline test/unit/hqmf/model/data_criteria_test.rb]: Should we complain if these mismatches are not found? I think that could only happen if we update data_criteria.rb to remove one of these fields.

**Review by c-monkey — APPROVED** [review]
Test passes.

File: test/unit/hqmf/model/data_criteria_test.rb (hunk 1 above)
Comment (edeyoung) [reply test/unit/hqmf/model/data_criteria_test.rb]: interesting idea! check out my change.

**Review by jbradl11 — APPROVED** [review]

**Comment by c-monkey** [comment]
I like the changed test.

**Pull request merged**